### PR TITLE
contains operator for fieldQuery and labelQuery

### PR DESCRIPTION
**Pull request by sigalmaya**
This pull request features support for a new operator "contains" operator. "contains operator is applicable for fieldQuery and labelQuery for get and delete APIs.
for example to retrieve all platforms that their name contains "service", use:
v1/platforms?fieldQuery="name contains 'service'"

### CODE REVIEW

File: pkg/query/operators.go [hunk 1]
```diff
@@ -25,6 +25,8 @@ const (
 	GreaterThanOperator gtOperator = "gt"
 	// GreaterThanOrEqualOperator takes two operands and tests if the left is greater than or equal the right
 	GreaterThanOrEqualOperator geOperator = "ge"
+	// Contains takes two operands and tests if the left contains the right
+	ContainsOPerator containsOperator = "contains"
```
Comment (alexwo) [inline pkg/query/operators.go:29]: should be ContainsOperator instead ContainsOPerator?

Comment (sigalmaya) [reply pkg/query/operators.go:29]: its O not null, it looks like null but is not

Comment (eyalnathan) [reply pkg/query/operators.go:29]:
```suggestion
	ContainsOperator containsOperator = "contains"
```

Comment (sigalmaya) [reply pkg/query/operators.go:29]: ahh sorry you mean P

**Review by dmitry-bakaleinik-sap — APPROVED** [review]

**Pull request merged**